package-lock.json

```diff
@@ -5,7 +5,8 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "2.1.0",
```

Comment:
Let's not introduce changes to this file

Reply:
Done